frontend/public/components/utils/managed-by.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+
+import { compareKinds, ResourceLink } from './resource-link';
+import { K8sResourceKind, referenceForOwnerRef } from '../../module/k8s';
+import { RootState } from '@console/internal/redux';
+import { withFallback } from '@console/shared/src/components/error/error-boundary';
+
+const managerStateToProps = (state: RootState): managerStateProps => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
+    receivedCSVs: state.k8s.getIn(['CSVS', 'csvs']),
+  };
+};
+
+const namespacedCSVManager = withFallback<CSVManagerProps>(
+  ({ activeNamespace, className, obj, receivedCSVs = [] }) => {
+    const ownerReferences = _.get(obj.metadata, 'ownerReferences') || [];
```

Comment:
Prefer optional chaining to `_.get` for better type safety

```suggestion
    const ownerReferences = obj?.metadata?.ownerReferences || [];
```